README.md

```diff
@@ -1 +1,50 @@
-# teleport-interview
\ No newline at end of file
+# teleport-interview
+
+Repository for the [Gravitational](https://goteleport.com/) [fullstack challenge](https://github.com/gravitational/careers/blob/master/challenges/fullstack/dashboard.pdf)
+
+## Requirements and Certificate Setup
+
+#### mkcert
+
+This project relies on self-signed certificates, which can cause your browser to complain at you and other minor annoyances. A simple tool for solving this is [mkcert](https://github.com/FiloSottile/mkcert), which creates a local CA that your browser will trust and uses it to sign subsequently generated certificates. Install `mkcert`, and then navigate to the `certs/` directory and generate a certificate and key for the app by running
+
+```bash
+# cd certs/
+mkcert -key-file localhost.key -cert-file localhost.crt 0.0.0.0 localhost 127.0.0.1 ::1
+```
```

Comment:
could you add a script to install mkcert and run this command (ideally dockerized), to reduce the steps needed to try this out?

Reply:
I think this could get tricky, since when you run `mkcert -install` the program needs to access your system trust store and add a CA cert/key. Possibly this could work in a Docker container, but then I think you would need to run the browser from that container in order for it to trust the CA.

I could experiment with it but frankly seems more trouble than its worth for this project.